CHANGELOG.md

```diff
@@ -4,9 +4,22 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 
 ## Next
 
+### New and Improved
+
```

Comment:
I'm confused. How does this work? When does `main` get `0.18.1` into the changelog?

Reply:
I'm... not sure actually. I guess I can update this PR.